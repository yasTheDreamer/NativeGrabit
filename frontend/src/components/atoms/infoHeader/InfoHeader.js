import React from 'react';
import {View, Text} from 'react-native';
import styles from './InfoHeaderStyle';

const InfoHeader = (props) => {
  return (
    <>
      <Text style={styles.title}>{props.title}</Text>
      <View style={styles.line}></View>
    </>
  );
};

export default InfoHeader;
