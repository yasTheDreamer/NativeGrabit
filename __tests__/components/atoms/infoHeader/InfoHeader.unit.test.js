import 'react-native';
import React from 'react';

import {render as enzymeRender, shallow} from 'enzyme';
import {render as reactRender} from '@testing-library/react-native';
import InfoHeader from '../../../../src/components/atoms/infoHeader/InfoHeader';

describe('InfoHeader component', () => {
  const wrapper = shallow(<InfoHeader title="test" />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('renders with the right title', () => {
    const {getByText} = reactRender(<InfoHeader title="test" />);

    expect(getByText('test')).toBeTruthy();
  });
});
