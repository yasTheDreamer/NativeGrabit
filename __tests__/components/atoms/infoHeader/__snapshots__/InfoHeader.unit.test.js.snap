// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoHeader component renders correctly 1`] = `ShallowWrapper {}`;
